src/main/java/com/shanebeestudios/skbee/elements/recipe/expressions/ExprRecipeWithId.java

```diff
@@ -0,0 +1,70 @@
+package com.shanebeestudios.skbee.elements.recipe.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import com.shanebeestudios.skbee.api.recipe.RecipeUtil;
+import org.bukkit.Bukkit;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Recipe - Recipe with Id")
+@Description({"Gets a minecraft recipe based off a given recipe id. Requires Minecraft 1.13+",
```

Comment:
```suggestion
@Description({"Gets a minecraft recipe based off a given recipe id.",
```
I got all the way down here and then realized the "requires mc 1.13" isn't needed, since this version won't work on Skript 1.13 anyways. 
Now I have to scroll up and change some stuff haha